test/aws_dynamodb_SUITE.erl

```diff
@@ -0,0 +1,138 @@
+    ok = delete_table(TableConf),
+    ok.
+
+all() ->
+    [put_get].
+
+%%--------------------------------------------------------------------
+%% TEST CASES
+
+put_get(Config) ->
+    TableConf = ?config(table_conf, Config),
+    put_item(TableConf, {1, 2}),
+    ?assertMatch(
+        {ok,
+            #{
+                <<"Item">> :=
+                    #{
+                        <<"TestHash">> := #{<<"S">> := <<"1">>},
+                        <<"TestRange">> := #{<<"S">> := <<"2">>}
+                    }
+            },
+            {200, _RespHeaders, _}},
+        get_item(TableConf, {1, 2})
+    ),
+    ok.
+
+%%--------------------------------------------------------------------
+%% HELPERS
+
+client() ->
+    aws_client:make_local_client(
+        <<"AccessKeyID">>, <<"SecretAccessKey">>, <<"8000">>, list_to_binary(os:getenv("DYNAMODB_HOST", "localhost"))
+    ).
+
+create_table(TableConf) ->
+    Client = client(),
+    Body = create_table_body(TableConf),
+    aws_dynamodb:create_table(Client, Body).
+
+put_item({TableName, _HashConf, _RangeConf} = TableConf, Item) ->
+    Client = client(),
+    Input = #{<<"TableName">> => TableName, <<"Item">> => mk_item(TableConf, Item)},
+    aws_dynamodb:put_item(Client, Input).
+
+get_item({TableName, {HashName, HashType}, {RangeName, RangeType}}, {HashKey, RangeKey}) ->
+    Client = client(),
+    Input = #{
+        <<"Key">> => #{
+            HashName => #{
+                HashType => HashKey
+            },
+            RangeName => #{
+                RangeType => RangeKey
+            }
+        },
+        <<"TableName">> => TableName
+    },
+    aws_dynamodb:get_item(Client, Input).
+
+create_table_body({TableName, {HashName, HashType}, {RangeName, RangeType}}) ->
+    #{
+        <<"AttributeDefinitions">> =>
+            [
+                #{
+                    <<"AttributeName">> => HashName,
+                    <<"AttributeType">> => HashType
+                },
+                #{
+                    <<"AttributeName">> => RangeName,
+                    <<"AttributeType">> => RangeType
+                }
+            ],
+        <<"KeySchema">> =>
+            [
+                #{
+                    <<"AttributeName">> => HashName,
+                    <<"KeyType">> => <<"HASH">>
+                },
+                #{
+                    <<"AttributeName">> => RangeName,
+                    <<"KeyType">> => <<"RANGE">>
+                }
+            ],
+        <<"ProvisionedThroughput">> =>
+            #{<<"ReadCapacityUnits">> => 5, <<"WriteCapacityUnits">> => 5},
+        <<"TableName">> => TableName
+    }.
+
+mk_item({_TableName, {HashName, HashType}, {RangeName, RangeType}}, {HashKey, RangeKey}) ->
+    #{
+        HashName => #{HashType => int_to_type(HashType, HashKey)},
+        RangeName => #{RangeType => int_to_type(RangeType, RangeKey)}
+    }.
+
+int_to_type(<<"S">>, I) -> integer_to_binary(I).
+
+delete_table(_) ->
+    %% fixme,
```

Comment:
How hard would it be to fix this? It doesn't seem to be needed though as I can run `rebar3 ct` locally several times in a row after setting up the docker container with: `docker run --rm -p 8000:8000 amazon/dynamodb-local:1.21.0` 

So my guess is that `create_table/1` actually returns an error the 2nd time around but since you don't match on the response, it's "ok" in a sense. 

I believe it'd be as simple as:
```suggestion
  {TableName, _HashConf, _RangeConf} = TableConf,
  aws_dynamodb:delete_table(client(), #{<<"TableName">> => TableName}).
```

Reply:
Thanks! I took both this and the create table suggestion.